[Bug]: WARNING: (x,y): warning CS0168: The variable 'e' is declared but never used

### Description

When importing the module on PowerShell 7, the following warning is shown.

```
WARNING: (544,45): warning CS0168: The variable 'e' is declared but never used
                        catch (COMException e)
```

This issue is just cosmetic.


### Steps to reproduce the issue

```PowerShell
WARNING: (544,45): warning CS0168: The variable 'e' is declared but never used
                        catch (COMException e)
```


### PowerShell Version

```Shell
PowerShell 7.4.1
```


### AutomatedLab Version

```Shell
5.51.0
```

--- a/AutomatedLabCore/internal/scripts/Types.ps1
+++ b/AutomatedLabCore/internal/scripts/Types.ps1
@@ -1093,7 +1093,7 @@ try
 }
 catch
 {
-    Add-Type -TypeDefinition $gpoType -IgnoreWarnings
+    Add-Type -TypeDefinition $gpoType -IgnoreWarnings 3>&1
 }
 
 try

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,7 @@
 - Fixed an issue in Stop-LabVm (#1619) with invalid error objects.
 - Fixed an issue in 'New-LWHypervVmConnectSettingsFile' storing
   the config file if the directory does not yet exist (#1611).
+- Suppressed a warning when adding the GPO .net type (#1627).
 
 ## 5.51.0 (2024-03-18)
 

